plop-templates/transition-component-test.hbs

```diff
@@ -0,0 +1,41 @@
+import { mount } from 'enzyme';
+
+import { checkAccessibility } from '../../../../test/util/accessibility';
+import {{name}} from '../{{name}}';
+
+describe('{{name}}', () => {
+  let container;
+
+  const create{{name}} = (props = {}) => {
+    const style = { width: 100, height: 200 };
+    return mount(
+      <{{name}} visible={false} {...props}>
+        <div style={style}>Test content</div>
```

Comment:
We can't inline the `style` object, because then there's nested curly braces (`style={{ width: 100, height: 200 }}`), making plop treat it as a template, and failing to parse it.

Reply:
In the first round of plop templates, we were still using JSDoc typing, so I had to figure this out. This appears to be a workaround.

```
 * {{!See https://github.com/handlebars-lang/handlebars.js/issues/1404 }}
 * @param { {{~name~}} Props} props
```